tests/screenshot/apps/components/Input.js

```diff
@@ -33,7 +34,18 @@ const InputTests = [
 	// Small input
 	<Input value="Simple value" size="small" />,
 	<Input value="Simple value" iconBefore="happyface" size="small" />,
-	<Input value="Simple value" iconAfter="happyface" size="small" />
+	<Input value="Simple value" iconAfter="happyface" size="small" />,
+
+	// Focus
+	...withConfig({focus: true}, [
+		<Input placeholder="Placeholder Input focused" style={{marginLeft: '3px'}} />,
+		<Input placeholder="Placeholder Input focused" disabled style={{marginLeft: '3px'}} />,
+		<Input value="Simple value focused" style={{marginLeft: '3px'}} />,
+		<Input value="Simple value focused" disabled style={{marginLeft: '3px'}} />,
+		<Input invalid invalidMessage="Custom invalid message focused" style={{marginLeft: '3px'}} />,
+		<Input value="Simple value focused" iconAfter="happyface" iconBefore="happyface" style={{marginLeft: '3px'}} />,
+		<Input value="Simple value focused" iconAfter="happyface" iconBefore="happyface" size="small" style={{marginLeft: '3px'}} />
```

Comment:
why we need margin left?

Reply:
I used marginLeft to differentiate the tests because they would fail otherwise, the tests would get compared to the unfocused ones.

Reply:
Ah, but I don't think so because you already change value or placeholder.   "Simple value focused" 
Could you check again?

Reply:
They still failed with "Simple value focused", but with "Focused simple value" it works. Thanks for the simple solution!